doc/Known_issues/CredSSP_error_bestiary.md

```diff
@@ -0,0 +1,28 @@
+# I get authentication error "CredSSP encryption Oracle remediation" when trying to RDP into a Windows machine from another Windows machine
+
+![CredSSP encryption Oracle remediation](CredSSPError.png)
+
+
+If you see the above error, please try the following two options:
+
+In this document the windows machine from where you are trying the RDP is called "client" and the
```

Comment:
Maybe this sentence should be moved before the one above, so that there's no 'hanging' colon

Reply:
ok